### ISSUE TITLE
Would any generated image samples be released?

### ISSUE THREAD

**Issue by naykun** [google-research/xmcgan_image_generation#11]
I understand that you probably cannot release pretrained model for now. But for a fair comparison between your methods and other future work, would you please release the generated samples on the validation set of LN-COCO, LN-OpenImage and COCO14?
